search-sentiments-map/public/scripts/trends-script.js

```diff
@@ -1,50 +1,47 @@
 // Copyright 2019 Google LLC
 //
-// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// Licensed under the Apache License, Version 2.0 (the 'License'); you may not
 // use this file except in compliance with the License. You may obtain a copy of
 // the License at
 //
 //     https://www.apache.org/licenses/LICENSE-2.0
 //
 // Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
 // WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 // License for the specific language governing permissions and limitations under
 // the License.
 
-/** Creates the world map. */
-function createWorldMap() {
-  map = new google.maps.Map(
-    document.getElementById('map'),
-    {center: {lat: 38.46049, lng: -5.428423}, zoom: 3});
-}
-
 const SHOW_MORE_OR_LESS_ID = 'show-more-or-less';
 const TOGGLE_SHOW_MORE = 'Show More';
 const TOGGLE_SHOW_LESS = 'Show Less';
 
 const CLASSNAME_SHOWN = 'shown';
 const CLASSNAME_HIDDEN = 'hidden';
+const NUM_SHOWN = 7;
 
-/** Displays the top trends of the US on the DOM. */
+/** Displays the top trends on the DOM. */
 function setTopTrends() {
   const trendsList = document.getElementById('trends-list');
 
-  // Get the top 20 trending search topics from the backend.
-  fetch('/trends').then(dailyTrendsJson => dailyTrendsJson.json()).then(dailyTrendsObj => {
-    var trendingSearches = dailyTrendsObj.default.trendingSearchesDays[0].trendingSearches;
+  // Get the top 10 globally trending search topics from the backend.
+  fetch('/trends').then(globalTrends => globalTrends.json()).then(trends => {
+    for (let i = 0; i < trends.length; i++) {
+      const trendElement = document.createElement('li');
+      trendElement.innerText = `${trends[i].trendTopic}`;
+      // Display the number of countries where the trend is trending when hovered.
+      trendElement.title = `Trending in ${trends[i].count} countries`;
 
-    for (var i = 0; i < trendingSearches.length; i++) {
-      var trendElement = document.createElement('li');
-      trendElement.innerText = trendingSearches[i].title.query;
-      // Show 10 trending topics by default and hide the rest.
-      trendElement.className = i < 10 ? CLASSNAME_SHOWN : CLASSNAME_HIDDEN;
+      // Show some trending topics by default and hide the rest.
```

Comment:
The number you use is fixed. Using the word "some" makes it sound like a random number of topics is displayed every time.

```suggestion
      // Show a certain number of trending topics by default and hide the rest.
```

Reply:
ok